templates/docker-compose/docker-compose.prod.yml.dist

```diff
@@ -41,6 +41,33 @@ services:
             - db_password
         command: 'bash -c "service cron start && tail -f /var/log/cron.log"'
 
+    supportpal_websockets:
+        container_name: '${WS_SERVICE_NAME}'
+        image: 'public.ecr.aws/supportpal/helpdesk:${APP_VERSION}-sockets-buster'
+        restart: always
+        depends_on:
+            - db
+            - redis
+            - mailer
+        environment:
+            REDIS_HOST: ${CACHE_SERVICE_NAME}
+            REDIS_PORT: ${REDIS_PORT}
+            HOST: ${HOST}
+            MAILER_SERVICE_NAME: ${MAILER_SERVICE_NAME}
+            WS_SERVICE_NAME: ${WS_SERVICE_NAME}
+        volumes:
+            - supportpal_resources:/supportpal/resources
+            - supportpal_app:/supportpal/app
+            - supportpal_config:/supportpal/config/production
+            - supportpal_storage:/supportpal/storage
+            - ./customization/:/customization
```

Comment:
This will need updating to account for https://github.com/supportpal/helpdesk-install/pull/46